source/language/minimal_profile.md

```diff
@@ -0,0 +1,130 @@
+# OpenQASM 3.0 `minimal` profile
+
+
+## Overview
+
+This *profile* defines a minimum subset of the OpenQASM 3.0 language
+that implementors are assumed to support. This should be sufficient to
+form a target for experiment authors or circuit libraries who intend
+to express *executable* OpenQASM -- that is, OpenQASM programs that
```

Comment:
Some philosophical discussion that we didn't quite settle:

1. Is a "base profile" the same as a "minimal profile", or do these suggest different things? How might this relate to future extensions or profiles?
2. Is a minimal profile for executable OpenQASM compatible with a minimal profile for e.g. theorists to describe quantum circuits?
3. To what extent should we work backwards from the state of existing open source tools in this profile?